analyzer/util/util.go

```diff
@@ -103,3 +103,13 @@ func SanitizeTxHash(hash string) *string {
 
 	return &hash
 }
+
+func TxHashToIndex(hash string, lookup map[string]int) *int {
+	index, exists := lookup[hash]
+
+	if exists {
+		return &index
+	}
+
+	return nil
```

Comment:
This should never happen, right? We should log some big fat errors here (though it's awkward that we don't have much context available ... the caller has it, really).